hydra/definitions.py

```diff
@@ -233,6 +235,26 @@ def NAF(x):
         Gx=0,
         Gy=0,
     ),
+    BANDERSNATCH_ID: Curve(
+        cairo_zero_namespace_name="bandersnatch",
+        id=BANDERSNATCH_ID,
+        p=0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001,
+        n=0x1cfb69d4ca675f520cce760202687600ff8f87007419047174fd06b52876e7e1,
+        h=4,
+        irreducible_polys={},
+        nr_a0=None,
+        nr_a1=None,
+        a=0x73EDA753299D7D483339D80809A1D80553BDA402FFFE5BFEFFFFFFFE1FB22001,  # See https://neuromancer.sk/std/other/Curve25519 for weirstrass form conversion utilities
+        b=0x73EDA753299D7D483339D80809A1D80553BDA402FFFE5BFEFFFFB870D2E00001,
```

Comment:
the a and b parameters seems wrong according to https://eprint.iacr.org/2021/1152.pdf section 3.1

Reply:
@feltroidprime 

`a` is −3763200000 mod 52435875175126190479447740508185965837690552500527637822603658699934817984513

same is with `b`

Reply:
right